source/rust.txt

```diff
@@ -37,35 +37,71 @@ See `Installation <https://github.com/mongodb/mongo-rust-driver#Installation>`__
 Connect to MongoDB Atlas
 ------------------------
 
+Select from the :guilabel:`Sync` or :guilabel:`Async` tabs below for
+corresponding connection code samples.
+
 .. include:: /includes/atlas-connect-blurb.rst
 
-.. code-block:: rust
+.. tabs::
+
+   .. tab:: Sync
+      :tabid: rust-sync
+
+      .. code-block:: rust
+
+         // Be sure to set default-features=false and enable the "sync" feature in your Cargo.toml
```

Comment:
This comment feels important - is there a page we can link to that covers `Cargo.toml` for the Rust driver? (Maybe not since it's new docs for a new driver)

Reply:
I think this is covered [here](https://github.com/mongodb/mongo-rust-driver#Installation), but maybe duplicating that or linking to that section would be useful in the tab. Will try something along those lines.